content/_layouts/frame.html.haml

```diff
@@ -169,6 +169,9 @@
                     %h5 Community
 
                     %ul.community
+                      %li
+                        %a{:href => 'https://community.jenkins.io'}
+                          Community
```

Comment:
```suggestion
                          Forum
```

Reply:
Yea, I don't have a better suggestion. Discuss was the only other idea and I don't know if thats clearer.